components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -9,7 +9,6 @@
 // - for cosmetic filters work with CSS and stylesheet. That work itself
 //   could call the script several times.
 
-const { parseDomain, ParseResultType } = require('parse-domain')
```

Comment:
I believe we can remove our `parse-domain` dependency from `package.json`, this is the only place it's used

Reply:
Removed, thanks!